### PR TITLE
move ICDS code to its own repo

### PR DESCRIPTION
**Pull request by snopoke**
New home of ICDS code: https://github.com/dimagi/commcare-icds
Changes:

* Delete `custom/icds`
* Delete `custom/icds_reports`
* Delete `custom/__init__.py` (to make it into a namespace package)
* Update `.travis.yml`

### CODE REVIEW

**Comment by snopoke** [comment]
build is now passing proving concept

**Comment by snopoke** [comment]
Deployed to `staging`

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +30,14 @@ install:
 before_script:
   - docker version
   - docker-compose version
+  # checkout the ICDS code from it's own repo
```
Comment (millerdev) [inline .travis.yml:33]: Is this expected to be a permanent part of test runs?

**Review by millerdev — APPROVED** [review]
:tada: :balloon:

File: .travis.yml (hunk 1 above)
Comment (snopoke) [reply .travis.yml:33]: For the time being yes. We don't have a way to run the ICDS tests outside of HQ at present. Also there's enough dependency between the two code bases that we want to make sure they continue to work together. 

I would hope we can change this in the future but for now this is necessary.

**Review by millerdev — APPROVED** [review]

**Comment by mkangia** [comment]
+326 −235,398

nothing can beat that!

**Comment by snopoke** [comment]
This branch got messed up due to some bad merges. replaced by https://github.com/dimagi/commcare-hq/pull/28301

**Pull request closed without merging**